<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Jayawork" failures="1" tests="3" name="Test" time="3.79" errors="0" timestamp="2025-02-10T08:13:16 CST">
  <testcase classname="day46.OrangeHRM" name="testLogo" time="0.184"/>
  <testcase classname="day46.OrangeHRM" name="testAppUrl" time="0.022">
    <failure type="java.lang.AssertionError" message="expected [https://opensource-demo.orangehrmlive.com/] but found [https://opensource-demo.orangehrmlive.com/web/index.php/auth/login]">
      <![CDATA[java.lang.AssertionError: expected [https://opensource-demo.orangehrmlive.com/] but found [https://opensource-demo.orangehrmlive.com/web/index.php/auth/login]
at day46.OrangeHRM.testAppUrl(OrangeHRM.java:37)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- testAppUrl -->
  <testcase classname="day46.OrangeHRM" name="testHomePageTitle" time="0.0">
    <skipped/>
  </testcase> <!-- testHomePageTitle -->
</testsuite> <!-- Test -->
